### PR TITLE
[2049] Activate end to end tests for funding

### PR DESCRIPTION
**Pull request by avinhurry**
### Context

We are switching on the funding feature flags for our existing end to end tests.

### Changes proposed in this pull request

- Switch on the funding feature flags for the existing end to end feature tests.
- Make the failing end to end feature tests pass by completing the funding section.

### Guidance to review

- Run the tests and hopefully watch them pass.

### CODE REVIEW

File: app/views/trainees/funding/training_initiatives/edit.html.erb [hunk 1]
```diff
@@ -13,7 +13,7 @@
       <h1 class="govuk-heading-l"><%= t("components.page_titles.trainees.funding.training_initiatives.edit") %><h1>
       <p class="govuk-body"><%= t("views.forms.funding.training_initiatives.description_html") %></p>
 
-        <%= f.govuk_radio_buttons_fieldset(:training_initiative, legend: { text: t("views.forms.funding.training_initiatives.#{training_initiative_options(@trainee).count == 2 ? 'title_two' : 'title'}"), size: "m" }) do %>
+        <%= f.govuk_radio_buttons_fieldset(:training_initiative, legend: { text: t("views.forms.funding.training_initiatives.#{training_initiative_options(@trainee).count == 2 ? 'title_two' : 'title'}"), size: "m" }, classes: "training-initiatives") do %>
```
Comment (FriedSock) [inline app/views/trainees/funding/training_initiatives/edit.html.erb:16]: Do we need to add this class? Can't we get away with just using `.govuk-radios` in the siteprism thing? I don't like adding stuff to the page just so we can select it in the tests.

File: spec/support/features/funding_details_steps.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Features
+  module FundingDetailsSteps
+    def and_the_funding_details_is_complete
+      review_draft_page.funding_section.link.click
+      edit_funding_page.training_initiative_radios.choose("Now teach")
```
Comment (FriedSock) [inline spec/support/features/funding_details_steps.rb:7]: Is "Now teach" a training initiative available for all routes? Maybe change this to
```suggestion
      edit_funding_page.training_initiative_radios.first.choose
```
to make it future proof for new routes

Comment (FriedSock) [reply spec/support/features/funding_details_steps.rb:7]: ah it is available for all routes 🤷

File: app/views/trainees/funding/training_initiatives/edit.html.erb (hunk 1 above)
Comment (avinhurry) [reply app/views/trainees/funding/training_initiatives/edit.html.erb]: All sorted

**Review by FriedSock — APPROVED** [review]

**Pull request merged**